services/account_type/cloudtrail_owner.yaml

```diff
@@ -30,7 +30,6 @@ Resources:
     Type: 'AWS::SQS::Queue'
     Condition: CreateResources
     Properties:
-      QueueName: !Sub 'cz-reactor-events-${ReactorAccountId}'
```

Comment:
Remove these for now ... let the stack auto gen. The prefix will be the Stack Name, which is at the customer's discretion so I don't think we need to be as conventional as we are with our own resources.

Reply:
I agree - as long as things are tagged and easy to see, that's good enough IMO